includes/class-wc-payments.php

```diff
@@ -841,4 +843,44 @@ function( string $cookie_hash ) {
 
 		return (int) $_SERVER['HTTP_X_WCPAY_PLATFORM_CHECKOUT_USER'];
 	}
+
+	/**
+	 * Remove default billing email field for Platform Checkout
+	 *
+	 * @param array $fields WooCommerce checkout fields.
+	 * @return array WooCommerce checkout fields.
+	 */
+	public static function platform_checkout_remove_default_email_field( $fields ) {
+		unset( $fields['billing']['billing_email'] );
```

Comment:
After the discussion on WooPay Sync meeting they raised the question if moving this field will break some theme/plugins. I was thinking we could add a check here to see if `$fields` has changed from it's default and then not doing anything with the email.